inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/deformable_convolution.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/deformable_convolution.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32, InferenceEngine::Precision::FP16};
```

Comment:
Same as before, we should test for integer precisions too

Reply:
Done [here](https://github.com/openvinotoolkit/openvino/pull/4312/commits/2fefe8926da65171f64a2dc679ab548415736213)

Reply:
Reverted since CPUFuncTest failed.

Reply:
Added again in [here](https://github.com/openvinotoolkit/openvino/pull/4312/commits/fc49c9e8a11798423904659ed99885f08cd8a7d0). This time CI passed so probably it was other issue not related to int types.